### DISCUSSION TITLE
Should I be backing up the machine learning cache

### DISCUSSION THREAD

**Discussion by RonaldJerez** [immich-app/immich#12059] [Q&A] [answered]
I'm trying to setup my backup strategy for unraid for immich.  I move my original photos to the array, and keep thumbnails on the cache.  But as I dont really understand what is in the machine learning cache directory I don't know if this is something I need to be backing up as well.  Initially I had it setup to do a nightly sync of the ML cache directory, but I noticed that each time is thousands of files that are constantly changing.

So my question is, what exactly is in the ML cache directory and should I be backing it up fully or partially for disaster recovery?  Or is this data that gets regenerated if I need to do a disaster recovery?

**Comment by bo0tzz** [comment, marked as answer]
The cache dir is just models, they get redownloaded if they're missing.

**Reply by RonaldJerez** [reply]
Awesome, that’s what I had suspected but wanted to make sure there was no facial recognition data being stored there that I may need to backup. 

Thanks!

**Comment by KecskeTech** [comment]
well yeah but in truenas it creates a temporary directory and lets say if my server starts everyday that aint good cuz it'll download like 10 gigs every day. Thanks for the answers, i'll create a dir in truenas.
